clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClient.java

```diff
@@ -801,11 +802,34 @@ static ZeebeClientCloudBuilderStep1 newCloudClientBuilder() {
   @ExperimentalApi("https://github.com/camunda/camunda/issues/20596")
   DecisionDefinitionQuery newDecisionDefinitionQuery();
 
+  /**
+   * Gets a decision definition by key.
+   *
+   * <pre>
+   *   long decisionDefinitionKey = ...;
+   *
+   *   zeebeClient
+   *   .newDecisionDefinitionGetRequest(decisionKey)
+   *   .send();
+   *   </pre>
+   *
+   * <p><strong>Experimental: This method is under development, and as such using it may have no
+   * effect on the client builder when called. The respective API on compatible clusters is not
+   * enabled by default. Thus, this method doesn't work out of the box with all clusters. Until this
+   * warning is removed, anything described below may not yet have taken effect, and the interface
+   * and its description are subject to change.</strong>
+   *
+   * @param decisionDefinitionKey the key of the decision definition
+   * @return a builder for the request to get a decision definition
+   */
+  @ExperimentalApi("https://github.com/camunda/camunda/issues/20596")
+  DecisionDefinitionGetRequest newDecisionDefinitionGetRequest(long decisionDefinitionKey);
+
   /*
    * Retrieves the XML representation of a decision definition.
    *
    * <pre>
-   * long decisionKey = ...;
+   * long decisionDefinitionKey = ...;
    *
    * zeebeClient
    *  .newDecisionDefinitionGetXmlRequest(decisionKey)
```

Comment:
```suggestion
   *  .newDecisionDefinitionGetXmlRequest(decisionDefinitionKey)
```

I am not sure if its decisionKey or decisionDefinitionKey here, I added suggestion considering its `decisionDefinitionKey :) `